Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -31,32 +30,50 @@ struct SwiftUINativeCheckoutDemo: View {
     }
 
     func checkoutView(_ title: String, _ content: String, _ isFlowComplete: Bool) -> some View {
-        NavigationView {
-            ZStack {
-                VStack(spacing: 16) {
-                    Picker("Checkout ", selection: $checkoutTypeSelection) {
-                        ForEach(CheckoutType.allCases) {type in
-                            Text(type.rawValue)
-                        }
+        VStack(spacing: 16) {
+            HStack {
+                Text("Checkout:")
+                    .font(.system(size: 18))
+                    .frame(alignment: .leading)
+                    .padding([.leading], 16)
+                    .padding([.trailing], 32)
+                Picker("", selection: $checkoutTypeSelection) {
+                    ForEach(CheckoutType.allCases) { type in
+                        Text(type.rawValue)
                     }
-                    Text(title)
-                    Text(content)
-                    Button(isFlowComplete ? "Try Again" : "Start Checkout" ) {
-                        if isFlowComplete {
-                            viewModel.retry()
-                        } else {
-                            startNativeCheckout()
-                        }
+                }.frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
nit: we mix both having this on a new line vs right after the curly brace. Looking through the apple SwiftUI docs it seems like newline is the preference. Should we do the same?